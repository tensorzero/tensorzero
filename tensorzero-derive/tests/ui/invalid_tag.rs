#[derive(tensorzero_derive::TensorZeroDeserialize)]
#[serde(tag = 123)]
enum Foo {
    Bar,
    Baz,
}

fn main() {}
