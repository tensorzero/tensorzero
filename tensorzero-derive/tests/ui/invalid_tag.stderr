error: TensorZeroDeserialize: #[serde(tag = "...")] attribute must be a string literal
 --> tests/ui/invalid_tag.rs:2:15
  |
2 | #[serde(tag = 123)]
  |               ^^^
